pkg/dev/coreos/Dockerfile

```diff
@@ -188,7 +188,7 @@ CMD [ "if [ -f /data/hubble_buildinfo ] ; then echo \"\" >> /hubble_build/hubble
 #possile to optionally include a custom one with the package
     && if [ -f /data/hubble ] ; then cp /data/hubble /etc/hubble/ ; fi \
 #also bring in anything from a /data/opt/ directory so we can bundle other executables if needed
-    && if [ -d /data/opt ] ; then cp -r /data/opt/* /hubble_build/debbuild/hubblestack-${HUBBLE_VERSION}/opt/ ; fi \
+    && if [ -d /data/opt ] ; then cp -r /data/opt/* /opt/ ; fi \
```

Comment:
If something is mounted in /data/opt folder, the Dockerfile fails because the current path does not exist. 
Perhaps it was mistakenly copied from some other Dockerfile.
Now I have made the dev Dockerfile of coreos similar to the pkg Dockerfile.
( I have also left you a comment on slack regarding this )

Reply:
👍